src/DSP.jl

```diff
@@ -24,6 +21,14 @@ end
 
 @importffts
 
+if VERSION >= v"0.7.0-DEV.602"
+    if isdefined(Base, :DSP)
+        import Base: conv, conv2, deconv, filt, filt!, xcorr
+    else
+        export conv, conv2, deconv, filt, filt!, xcorr
+    end
+end
+
 include("dspbase.jl")
```

Comment:
@ararslan Do you think this fix will get things working? Or at least not cause extra problems on 0.7 while getting things working on 0.6?